test/commonControlTests/NumberControl.spec.ts

```diff
@@ -59,6 +59,32 @@ suite('NumberControl e2e tests', () => {
                 (requestHandler.getSerializableControlStates()[TEST_CONTROL_ID] as NumberControlState).value,
             ).eq(16);
         });
+
+        test('APL event, no confirmation, no validation required', async () => {
+            const requestHandler = new ControlHandler(new NumberControlManager());
+            await testE2E(requestHandler, [
+                'U: ',
+                TestInput.of(GeneralControlIntent.of({})),
+                'A: What number?',
+                'U: ',
+                TestInput.userEvent({
+                    type: 'Alexa.Presentation.APL.UserEvent',
+                    requestId: 'amzn1.echo-api.request.1',
+                    timestamp: '2020-11-30T18:48:22Z',
+                    locale: 'en-US',
+                    token: '',
+                    arguments: ['NumberSelectorWithoutValidationExpectation', 15],
+                    source: {
+                        type: 'EditText',
+                        id: 'editTextNumber',
+                    },
+                }),
+                'A: Ok. Value set to 15.',
+            ]);
+            expect(
+                (requestHandler.getSerializableControlStates()[TEST_CONTROL_ID] as NumberControlState).value,
```

Comment:
no problem here.. but we do need to find a better way to test these things.  One idea is to make ControlManager stateful so that we can do things like myControlManager.getRootControl() / .getControlById() etc.